src/components/slide-toggle/slide-toggle.html

```diff
@@ -0,0 +1,9 @@
+<div class="md-container">

```

Comment:
Can you rename the CSS classes so that they're more explicitly for the slide-toggle? E.g., 
`md-slidetoggle-container`